test/e2e/util.go

```diff
@@ -177,6 +177,10 @@ func Logf(format string, a ...interface{}) {
 	fmt.Fprintf(GinkgoWriter, "INFO: "+format+"\n", a...)
 }
 
+func Errorf(format string, a ...interface{}) {
+	fmt.Fprintf(GinkgoWriter, "EROR: "+format+"\n", a...)
```

Comment:
We have a bunch of other places in the e2e tests that do an info log that says "failed to do X" which is probably sufficient for the four places you were using this function.